### DISCUSSION TITLE
Title Bar & Window Outline Showing

### DISCUSSION THREAD

**Discussion by jetski28** [OpenSurv/OpenSurv#62] [General]
I've just setup my first OpenSurv to test out and while it appears to do what I need the Title Bar & Window Border shows for each stream. This is causing streams to overlap blocking some of the stream above it. Have I missed something? The demo feeds did the same.

![image](https://github.com/user-attachments/assets/f559dbe3-106c-429a-8906-0b36ee1d7ae8)

**Comment by SvenVD** [comment]
This should not happen, what OS and hardware are you running

**Comment by SvenVD** [comment]
I guess you are not using Ubuntu 24.04 LTS? You can try with Ubuntu 24.04 LTS then it should work.

**Comment by jetski28** [comment]
No I wasn't - I was trying Debian 12.
The hardware list / OS requirements weren't overly clear other than a "discussion".

I will give Ubuntu 24.04 a try.

**Comment by jointine-john** [comment]
I'm not seeing experiencing this issue, I'm running Debian 12 on an Intel Nuc,

I installed Debian GUI less and let the installer add the UI, I had to manually install rsync, and python3-yaml. and re run the installer, but it all seems fine for me.

**Comment by rob196** [comment]
I have just run into this same issue, any ideas what I could maybe try?

- OS: Ubuntu 24.04.1 LTS
- Hardware: GMKtec Nucbox G3 Plus (Intel N150, 16GB RAM)

**Comment by SvenVD** [comment]
> I have just run into this same issue, any ideas what I could maybe try?
> 
> * OS: Ubuntu 24.04.1 LTS
> * Hardware: GMKtec Nucbox G3 Plus (Intel N150, 16GB RAM)

Did you do anything special before or after running the installer?

**Comment by rob196** [comment]
> > I have just run into this same issue, any ideas what I could maybe try?
> > 
> > * OS: Ubuntu 24.04.1 LTS
> > * Hardware: GMKtec Nucbox G3 Plus (Intel N150, 16GB RAM)
> 
> Did you do anything special before or after running the installer?

Only things I did after Ubuntu install was enable SSH, then installed OpenSurv and then updated the monitor files and restarted the service (tried a reboot as well)

**Comment by IxFail** [comment]
Had the same issue on my test machine, with Ubuntu 24.04.1 LTS. based on [the cloudimage of ubuntu](https://cloud-images.ubuntu.com/noble/current/noble-server-cloudimg-amd64.ova)
Found out that xfwm4 was the issue.
Solved it with the following steps as a superuser:

1. `mv /usr/share/themes/Default /usr/share/themes/Default-bck`
2. `mkdir -p /usr/share/themes/Default/xfwm4/`
3. `touch /usr/share/themes/Default/xfwm4/themerc`
4. `service lightdm restart`

Problem is gone.
What the above commands do:

- Makes a backup of the Default theme
- Make an empty theme
- Restart the lightdm service

Reference for the empty theme solution: [https://forum.xfce.org/viewtopic.php?pid=74577#p74577](https://forum.xfce.org/viewtopic.php?pid=74577#p74577)

**Comment by rob196** [comment]
> Had the same issue on my test machine, with Ubuntu 24.04.1 LTS. based on [the cloudimage of ubuntu](https://cloud-images.ubuntu.com/noble/current/noble-server-cloudimg-amd64.ova) Found out that xfwm4 was the issue. Solved it with the following steps as a superuser:
> 
> 1. `mv /usr/share/themes/Default /usr/share/themes/Default-bck`
> 2. `mkdir -p /usr/share/themes/Default/xfwm4/`
> 3. `touch /usr/share/themes/Default/xfwm4/themerc`
> 4. `service lightdm restart`
> 
> Problem is gone. What the above commands do:
> 
> * Makes a backup of the Default theme
> * Make an empty theme
> * Restart the lightdm service
> 
> Reference for the empty theme solution: https://forum.xfce.org/viewtopic.php?pid=74577#p74577

Perfect, thanks, this solved my issue

**Comment by SvenVD** [comment]
If you get your ubuntu 24.04 from https://ubuntu.com/download/desktop does that still have the issue?

**Comment by rob196** [comment]
> If you get your ubuntu 24.04 from https://ubuntu.com/download/desktop does that still have the issue?

Re-downloaded the ISO from above link

Flashed with balenaEtcher (previously used rufus)

- During install, this time I got an update message and I so I updated the installer before continuing (after update needed to re-initialize the installer)
- I chose:
  - Interactive Installation
  - Default Selection
  - Selected Download & Install support for additional media formats
  - Erase disk & Install Ubuntu
  - Rebooted
- Installed SSH
- Ran the Ubuntu Software Updater and updated what was listed)
- Rebooted
- Via ssh session:
  - Installed git
  - Installed OpenSurv (all default)
  - Rebooted

It shows the Title bar & window

![Image](https://github.com/user-attachments/assets/d80bee8b-ef72-496a-8d71-5dbddd76a522)

**Comment by SvenVD** [comment]
can you try adding:

` freeform_advanced_mpv_options: "--script-opts=osc-windowcontrols=no"` to the stream to see if this solves the issue for your case? What is the mpv version you are  running?

Also what does `ps -ef | grep mpv` output gives?

**Comment by SvenVD** [comment]
stale, closing
